modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticOrthotropic.h

```diff
@@ -0,0 +1,66 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef TENSORMECHANICSPLASTICORTHOTROPIC_H
+#define TENSORMECHANICSPLASTICORTHOTROPIC_H
+
+#include "TensorMechanicsPlasticIsotropicSD.h"
+
+class TensorMechanicsPlasticOrthotropic;
+
+template<>
+InputParameters validParams<TensorMechanicsPlasticOrthotropic>();
+
+/**
+ * Orthotropic plasticity model from Yoon (2013)
+ * the name of the paper is "Asymmetric yield function based on the
+ * stress invariants for pressure sensitive metals" published
+ * 4th December 2013.
+ * This model accounts for sensitivity in pressure, the
+ * strength differential effect and orthotropic behavior
+ * Yield_function = b*I1 + (J'2^{3/2} - c*J''3)^{1/3} - yield_strength
+ * The last three functions are the main functions that call all other
+ * functions in this module for the Newton-Raphson method.
+ */
+class TensorMechanicsPlasticOrthotropic : public TensorMechanicsPlasticIsotropicSD
+{
+public:
+  /**
+   * The yield function for Orthotropic needs the the constants _b, _c1 and _c2
+   * the bool _associative decides if the it uses the associative flow rule
+   * or non-associative flow rule with the derivatives of the von Mises yield function.
+   *
+   * @real _b, A constant to model the influence of pressure
+   * @real _c1, The six coefficients of L prime
+   * @real _c2, The six coefficients of L prime prime
+   * @bool _associative, Flag for flow-rule, true if not specified
+   */
```

Comment:
This is not how the Doxygen documentation works. Check out code that we already have in MOOSE for valid examples. `@real` is not a thing.